src/components/chatbox/chatbox.module.css

```diff
@@ -21,6 +21,12 @@
 }
 
 .arrowdown {
+	/* TODO(kevinwang): 
+	 * 1. consider passing size="small" to <Button/>;
+	 * 2. figure out compromise between the design spec (square button)
+	 * and HDS, where a perfectly square button is not yet supported
+	 * (https://helios.hashicorp.design/components/button?tab=code#icon-only-button) */
```

Comment:
I think we are good to just use `small` here and remove this override. @kellyschummer thoughts? Re: the HDS button convo https://hashicorp.slack.com/archives/C059QUX1AUA/p1696368281165159

Reply:
Small works well here- since it's a secondary support element.